src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -122,9 +108,10 @@ protected override OperationStatus<IMethodSymbol> GenerateMethodDefinition(bool
                         Formatter.Annotation.AddAnnotationToSymbol(methodSymbol)));
             }
 
-            protected override async Task<SyntaxNode> GenerateBodyForCallSiteContainerAsync(CancellationToken cancellationToken)
+            protected override async Task<SyntaxNode> GenerateBodyForCallSiteContainerAsync(
+                SyntaxNode container,
+                CancellationToken cancellationToken)
             {
-                var container = GetOutermostCallSiteContainerToProcess(cancellationToken);
```

Comment:
both teh caller and this function needed the 'container'.  so cleaner and safer to pass it in so that both are talking about the same object.